components/plurals/src/provider.rs

```diff
@@ -129,6 +128,21 @@ pub enum RawPluralCategory {
     Many = 5,
 }
 
+impl RawPluralCategory {
+    /// Gets the corresponding variant string of this `RawPluralCategory`.
+    #[cfg(any(feature = "datagen", feature = "serde"))]
+    const fn as_str(self) -> &'static str {
```

Comment:
Nit: make these lowercase like they are in ECMA-402.

On the enum, add this attribute: `#[serde(rename_all = "lowercase")]`

Then down here, use lowercase strings.